js/explore/force_dependencyGraph.js

```diff
@@ -0,0 +1,750 @@
+                .attr('y1', d => d.source.y)
+                .attr('x2', d => d.target.x)
+                .attr('y2', d => d.target.y);
+
+            node.selectAll('circle')
+                .attr('cx', d => d.x)
+                .attr('cy', d => d.y);
+        });
+
+        // Data for legend
+        const labels = ['LLNL Repositories with Dependencies', 'External Packages', 'Internal Packages'];
+    
+        // Creates legend
+        const legend = chart
+            .append('g');
+
+        function updateLegend(labels, color = colors) {
+            legend.selectAll('g').remove();
+
+            const legendMap = [];
+            color.forEach((d, i) => {
+                legendMap.push({ text: labels[i], color: d });
+            });
+    
+            const legendEntries = legend
+                .selectAll('g')
+                .data(legendMap)
+                    .join('g')
+                    .attr('class', 'legend')
+                    .attr('transform', (d, i) => {
+                        const legendHeight = legendRectSize + legendSpacing;
+                        const offset = (legendHeight * colors.length) / 2;
+                        const horizontal = 0 - width / 2;
+                        const vertical = i * legendHeight - height / 2;
+                        return `translate(${horizontal}, ${vertical})`;
+                    });
+            
+            // Adds rectangle for color reference
+            legendEntries
+                .append('rect')
+                    .attr('width', legendRectSize)
+                    .attr('height', legendRectSize)
+                    .style('fill', d => {
+                        return d.color;
+                    })
+                    .style('stroke', d => {
+                        return d.color;
+                    });
+    
+            // Adds legend text
+            legendEntries
+                .append('text')
+                    .attr('x', legendRectSize + legendSpacing)
+                    .attr('y', legendRectSize - legendSpacing)
+                    .text(d => {
+                        return d.text;
+                    })
+                    .attr('text-anchor', 'start');
+        }
+
+        updateLegend(labels);
+
+        const options = {};
+
+        // Options for graph view
+        options.normalView = { name: 'normalView', text: 'Repos connected to dependencies', labels: ['LLNL Repositories with Dependencies', 'External Packages', 'Internal Packages'], function: redraw };
+        options.simplifiedView = { name: 'simplifiedView', text: 'Repos connected by shared dependencies', labels: ['LLNL Repositories with Dependencies', 'External Packages', 'Internal Packages'], function: simplify };
+        options.orgView = { name: 'orgView', text: 'Organizations connected to verified dependecy organizations', labels: ['LLNL Organizations', 'External Package Organizations', 'LLNL Package Organizations'], function: organize };
```

Comment:
`dependecy` needs a typo fix!

Additionally, is the graph still displaying only verified organizations? If not, this label should reflect that.